js/views/search.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+<layout :title="_('Search')">
+    <div class="row">
+        <datasets class="col-xs-12" :datasets="datasets" :title="datasetsTitle">
+        </datasets>
+    </div>
+    <div class="row">
+        <communities class="col-xs-12" :communities="communities" :title="communitiesTitle">
+        </communities>
+    </div>
+    <div class="row">
+        <reuses class="col-xs-12" :reuses="reuses" :title="reusesTitle">
+        </reuses>
+    </div>
+    <div class="row">
+        <issues class="col-xs-12" :issues="issues" :title="issuesTitle">
+        </issues>
+    </div>
+    <div class="row">
+        <discussions class="col-xs-12" :discussions="discussions" :title="discussionsTitle">
+        </discussions>
+    </div>
+</layout>
+</template>
+
+<script>
+import {PageList} from 'models/base';
+import Layout from 'components/layout.vue';
+
+export default {
+    name: 'SearchView',
+    components: {
+        datasets: require('components/dataset/list.vue'),
+        communities: require('components/communityresource/list.vue'),
+        reuses: require('components/reuse/list.vue'),
+        issues: require('components/issues/list.vue'),
+        discussions: require('components/discussions/list.vue'),
+        Layout
+    },
+    data() {
+        return {
+            datasetsTitle: '',
+            datasets: new PageList({
+                ns: 'me',
+                fetch: 'my_org_datasets'
+            }),
+            communitiesTitle: '',
+            communities: new PageList({
+                ns: 'me',
+                fetch: 'my_org_community_resources'
+            }),
+            reusesTitle: '',
+            reuses: new PageList({
+                ns: 'me',
+                fetch: 'my_org_reuses'
+            }),
+            issuesTitle: '',
+            issues: new PageList({
+                ns: 'me',
+                fetch: 'my_org_issues',
+            }),
+            discussionsTitle: '',
+            discussions: new PageList({
+                ns: 'me',
+                fetch: 'my_org_discussions'
+            }),
+        };
+    },
+    attached() {
+        this.update();
+        this._handler = this.$root.me.$on('updated', this.update.bind(this));
+    },
+    detached() {
+        this._handler.remove();
+    },
+    methods: {
+        update() {
+            if (this.$root.me.id) {
```

Comment:
You can remove this line (User id is not needed as it's resolved server-side).
But you can trigger search update on route change:
```javascript
route: {
        data() {
            this.update();
        }
    },
```
Or, given it's updated only on route change, you can directly move the update code into route.data()